includes/CLI.php

```diff
@@ -121,10 +121,16 @@ public static function orders( $args, $assoc_args ) {
 	public static function customers( $args, $assoc_args ) {
 		list( $amount ) = $args;
 
+		$domain = '';
+
+		if ( ! empty( $assoc_args['email-domain'] ) ) {
+			$domain = $assoc_args['email-domain'];
+		}
```

Comment:
This can be consolidated to one line:

```suggestion
		$domain = $assoc_args['email-domain'] ?? '';
```